resources/xip-patch/templates/clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: xip-patch
+rules:
+- apiGroups: ["*"]
+  resources: ["configmaps", "services"]
+  verbs: ["*"]
```

Comment:
You don't need write access to `service`. Split it into two rules: one with read-write access to `configmaps` and second with read-only access to `services`.